scoreboard/controllers.py

```diff
@@ -82,6 +82,31 @@ def register_user(email, nick, password, team_id=None,
                     nick, email, flask.request.remote_addr)
     return user
 
+@utils.require_not_started
+def change_user_team(uid, team_tid, code):
+    """Provide an interface for changing a user's team"""
+    team = models.Team.query.get_or_404(team_tid)
+    user = models.User.query.get_or_404(uid)
+
+    old_team = user.team
+
+    if code.lower() != team.code.lower():
+        raise errors.ValidationError('Invalid team selection or team code')
+
+    if team.tid == user.team_tid:
+        raise errors.ValidationError('Changing to same team')
+
+    app.logger.info('User %s switched to team %s from %s' % (user.nick, team.name, old_team.name))
+    user.team = team
+    user.team_tid = team_tid
+    flask.session['team'] = team_tid
```

Comment:
If players keep their old cookies around, they'd be in position of having access to multiple teams... I think this is fine though.. let me think.